tests/data/annotations/rawvideo_test_anno.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "video_dir": "rawvideo_dataset",
```

Comment:
What's the difference between rawvideo and video? I think it should be consistent of dataset types.

Reply:
RawVideo dataset is used in the Project OmniSource. They have some differences.